packages/pyright-internal/src/common/pythonVersion.ts

```diff
@@ -76,7 +81,14 @@ export namespace PythonVersion {
         return true;
     }
 
-    export function isGreaterThan(version: PythonVersion, other: PythonVersion) {
+    export function isGreaterThan(version: PythonVersion | undefined, other: PythonVersion | undefined) {
```

Comment:
I'm a little uncomfortable defining boolean comparison operators for non-existent Python versions. Isn't the correct answer "undefined" in this case? Or are you thinking that a non-existent version is equivalent to a "negative" version and is therefore always considered less than all existent versions?

Reply:
That was my thought. Undefined version is always less than a defined version.